Tests/GRPCTests/GRPCClientStateMachineTests.swift

```diff
@@ -92,6 +92,58 @@ extension GRPCClientStateMachineTests {
     )).assertSuccess()
   }
 
+  func testSendingDefaultContentypeRequestHeader() {
+    var stateMachine = self
+      .makeStateMachine(.clientIdleServerIdle(pendingWriteState: .one(), readArity: .one))
+    stateMachine.sendRequestHeaders(requestHead: .init(
+      method: "POST",
+      scheme: "http",
+      path: "/echo/Get",
+      host: "host",
+      deadline: .distantFuture,
+      customMetadata: [:],
+      encoding: .disabled
+    )).assertSuccess { headers in
+      XCTAssertEqual(headers.first(name: "content-type"), "application/grpc")
+    }
+  }
+
+  func testSendingProtoContentypeRequestHeader() {
```

Comment:
nit: let's have a helper test which takes a `ContentType`, something like:

```swift
func doTestCustomContentType(_ contentType: ContentType) {
  var stateMachine = ...
  stateMachine.sendRequestHeaders(requestHead: .init(
    ...
    customMetadata: ["content-type": contentType.canonicalValue]
    ...
  )).assertSuccess { 
    ...
  } 
}
```

then we can just call that from our tests with `.flatbuffers` and `.protobuf`